department-of-veterans-affairs/abd-vro
# pull bip out of app (squashed)

## What was the problem?
bip -related classes scattered between rrd domain and app

Associated tickets or Slack threads:
[
- [MCP-000](https://amida.atlassian.net/browse/MCP-000)](https://github.com/orgs/department-of-veterans-affairs/projects/871/views/8?pane=issue&itemId=28826934)
- https://github.com/department-of-veterans-affairs/abd-vro/issues/1534

## How does this fix it?[^1]
All the bip-related stuff is moved to `svc-bip-api`.  Code that was inside of bip that didn't belong there was moved elsewhere.  

## How to test this PR
`./gradlew test`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
